apps/openmw/mwsound/soundmanagerimp.cpp

```diff
@@ -368,22 +368,35 @@ namespace MWSound
 
             mMusicFiles[mCurrentPlaylist] = filelist;
 
+            // Build mMusicToPlay for this playlist
+            tracklist.resize(filelist.size());
+            std::iota(tracklist.begin(), tracklist.end(), 0);
```

Comment:
Aren't these two lines redundant, since we do the exact same thing later on line 384?